src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/HealthCertificate.swift

```diff
@@ -68,7 +68,7 @@ struct HealthCertificate: HealthCertificateData, Codable, Equatable, Comparable
 	}
 
 	var dateOfBirthDate: Date? {
-		return Self.dateFormatter.date(from: digitalGreenCertificate.dateOfBirth)
+		return ISO8601DateFormatter.contactDiaryFormatter.date(from: digitalGreenCertificate.dateOfBirth)
```

Comment:
and I think in your other PR #2753 this will not longer name contactDiaryFormatter, right?

Reply:
Yep 👍🏼